profanity.rb

```diff
@@ -799,7 +799,7 @@ def get_color_pair_id(fg_code, bg_code)
 	str = str.gsub('lines', Curses.lines.to_s).gsub('cols', Curses.cols.to_s)
 	str.untaint
 	begin
-		proc { $SAFE = 3; eval(str) }.call.to_i
+		proc { $SAFE = 1; eval(str) }.call.to_i
```

Comment:
I am completely in favor of getting rid of it myself as well, as it adds a false sense of security that is still circumventable by anyone with the knowledge. It should really be removed from Lich.rb IMO as well, since most people willy-nilly `;trust` things without actually reading the source-code anyway.

I just wanted to make as few changes as possible to make sure Profanity works, and add the feature I originally opened it up to contribute.